src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -265,11 +257,25 @@ public void CopyRuntimeSettingsTo(JsonPropertyInfo other)
             other._escapedName = _escapedName;
         }
 
+        // Create a property that is either ignored at run-time. It uses typeof(int) in order to prevent
+        // issues with unsupported types and helps ensure we don't accidently (de)serialize it.
+        public static JsonPropertyInfo CreateIgnoredPropertyPlaceholder(PropertyInfo propertyInfo, JsonSerializerOptions options)
+        {
+            JsonPropertyInfo jsonPropertyInfo = new JsonPropertyInfoNotNullable<int, int, int>();
```

Comment:
Would this end up creating a new `JsonPropertyInfoNotNullable` every time? Should this be a static field which is initialized once and re-used for all ignored properties?

Reply:
It needs to have its name set, so it can't be a global.

Reply:
Ah right. Good point.